java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java

```diff
@@ -740,6 +740,39 @@ private static String getBusId(Terminal t) {
                             .addStringSeries("bus_id", svc -> getBusId(svc.getTerminal())))
                             .build();
 
+                case VOLTAGE_LEVEL:
+                    List<VoltageLevel> voltageLevels = network.getVoltageLevelStream().collect(Collectors.toList());
+                    return addProperties(new SeriesPointerArrayBuilder<>(voltageLevels)
+                            .addStringSeries("id", VoltageLevel::getId)
+                            .addDoubleSeries("nominal_v", VoltageLevel::getNominalV)
+                            .addDoubleSeries("high_voltage_limit", VoltageLevel::getHighVoltageLimit)
+                            .addDoubleSeries("low_voltage_limit", VoltageLevel::getLowVoltageLimit))
+                            .build();
```

Comment:
We should add the substation ID too, to be able to link VL -> substation -> country for example